docs/typescript.md

```diff
@@ -14,7 +14,7 @@ If you're starting a new project, there are a few different ways to get started.
 You can use the [TypeScript template][ts-template]:
 
 ```shell
-npx react-native init MyApp --template react-native-template-typescript
+npx react-native --ignore-existing init MyApp --template react-native-template-typescript
```

Comment:
I don't think this has to included in the official docs.

`--ignore-existing` is the symptom of a faulty environment (e.g. you installed globally packages while you should have not). If we do so, all the `npx` commands should also be adapted to include the same flag.

It's more of an instruction for a troubleshooting section

Reply:
May be we can add this as an extra instruction in case someone is facing this issue. 'Coz a lot of people off-late have been facing this issue.